README.md

```diff
@@ -119,6 +125,46 @@ http://localhost:3000/results
 ```
 
 ##### Matches
+
+###### All Matches
+
+```js
+app.get('/allMatches', (req, res) => {
```

Comment:
I'd prefer using `/all-matches` and encourage best practices, or rather more common solution.